app/templates/mobile.hbs

```diff
@@ -0,0 +1,12 @@
+{{page-title "Mobile"}} 
+<div class='qr-code-container'> 
+ <QrCode
+    data-test="qr-code"
+    @text= {{@model.userId}}
+    @darkColor='#00000'
+    class = 'qr-code'
+  />
+ </div>
+ <p class='qr-code-text'>Check out our RDS mobile app</p>
```

Comment:
Why this is being added at multiple places,
- We have component for the same we can simply call that
- It's in `index.hbs` as well, can we please remove it

Reply:
Here, `Qrcode` component can be called which you have created and then line 2 to 10 can be removed